HW8/marian_hrychak/task_1/models/__init__.py

```
@@ -0,0 +1,4 @@
+from .user import *
```

Ruff (0.8.2)

`from .user import *` used; unable to detect undefined names

(F403)